src/wiktextract/config.py

```diff
@@ -145,13 +173,14 @@ def set_attr_from_json(
     def init_subtitles(self) -> None:
         self.set_attr_from_json("LINKAGE_SUBTITLES", "linkage_subtitles.json")
         self.set_attr_from_json("POS_SUBTITLES", "pos_subtitles.json")
-        self.POS_TYPES = set(x["pos"] for x in self.POS_SUBTITLES.values())
-        for k, v in self.POS_SUBTITLES.items():
-            if "tags" in v:
-                assert isinstance(v["tags"], (list, tuple))
+        if self.POS_SUBTITLES is not None:
+            self.POS_TYPES = set(x["pos"] for x in self.POS_SUBTITLES.values())
+            for k, v in self.POS_SUBTITLES.items():
+                if "tags" in v:
+                    assert isinstance(v["tags"], (list, tuple))
```

Comment:
`set_attr_from_json` set the attribute to empty dictionary if the JSON file doesn't exist, so there we don't need to check `POS_SUBTITLES` is `None`. And the `assert` is also unnecessary because `json.load()` can't create `tuple`.